chaviprom/secure_otp_views.py

```
@@ -0,0 +1,328 @@
+from django.views.generic import View
+from django_otp import match_token
+from django_otp.decorators import otp_required
+from django import forms
+from django_ratelimit.decorators import ratelimit
+
+from two_factor.views import LoginView
+from two_factor.views.core import SetupView as BaseSetupView
+from django.contrib.auth import views as auth_views
+
+# Rate-limited Login View
+from django.utils.decorators import method_decorator
```

Ruff (0.12.2)

Redefinition of unused `method_decorator` from line 22

Remove definition: `method_decorator`

(F811)